src/widgets/dialogs/SettingsDialog.cpp

```diff
@@ -63,7 +85,7 @@ void SettingsDialog::initUi()
                     .withoutMargin()
                     .emplace<QLineEdit>()
                     .assign(&this->ui_.search);
-    edit->setPlaceholderText("Find in settings... (Ctrl+F)");
+    edit->setPlaceholderText("Find in settings... (default Ctrl+F)");
```

Comment:
@Felanbird I think this sounds better, not sure though. Trying to find absolutely every nitpick possible.
```suggestion
    edit->setPlaceholderText("Find in settings... (Ctrl+F by default)");
```